README.md

```diff
@@ -1,2 +1,13 @@
 # muhc-ui
-UI for MUHC Project
+## Purpose 
+
+Welcome to the frontend of MUHC microservices suite. This project is the sole frontend client for the web app of the MUHC project by this GitHub organization. 
```

Comment:
PRBN-UI suite is the sole frontend client for the web app of the  PRBN project by this Github organization